__tests__/intergration.test.js

```diff
@@ -57,6 +58,29 @@ describe('GET /api/articles/:article_id', () => {
         expect(body.article).toEqual(output);
       });
   });
+  test('get 200: should return an article as specified by its ID', () => {
```

Comment:
Ideally this would have been a good thing to write a whole new test for. However, I can see that this would have made your test here fail - so it may be worth considering flexible matchers like `toMatchObject` in future :)